cmd/kubeadm/app/phases/addons/proxy/proxy.go

```diff
@@ -144,5 +151,39 @@ func createClusterRoleBindings(client clientset.Interface) error {
 				Namespace: metav1.NamespaceSystem,
 			},
 		},
+	}); err != nil {
+		return err
+	}
+
+	// Create a role for granting read only access to the kube-proxy component config ConfigMap
+	if err := apiclient.CreateOrUpdateRole(client, &rbac.Role{
```

Comment:
Do we have tests to validate the updated RBAC rules?

Reply:
The only test on RBAC I'm aware of are in `test\e2e_kubeadm` but I'm not sure if those test are used in test grid batches...